File: backend/test_env.ts
import dotenv from 'dotenv';
import path from 'path';
import fs from 'fs';

console.log("Current directory:", process.cwd());
const envPath = path.resolve(process.cwd(), '../.env');
console.log("Looking for .env at:", envPath);
console.log("Exists?", fs.existsSync(envPath));

dotenv.config({ path: envPath });
console.log("GOOGLE_API_KEY:", process.env.GOOGLE_API_KEY ? "Found" : "Not Found");
console.log("JWT_SECRET:", process.env.JWT_SECRET ? "Found" : "Not Found");

File: ai-agent/test_env.py
import os
from dotenv import load_dotenv, find_dotenv

print(f"Searching for .env from: {os.getcwd()}")
dotenv_path = find_dotenv()
print(f"Found .env at: {dotenv_path}")

load_dotenv(dotenv_path)
key = os.getenv("GOOGLE_API_KEY")

if key:
    print(f"GOOGLE_API_KEY found: {key[:5]}...{key[-5:]}")
else:
    print("GOOGLE_API_KEY NOT found")
